src/OrdersList/Orders.cpp

```diff
@@ -4,63 +4,80 @@
 
 using namespace std;
 
-int Order::countOrderID = 0;        //start counter for orders at 0
+int* Order::countOrderID = new int(0);        //start counter for orders at 0
```

Comment:
how come ur counter is a pointer? the assigment states that all `user defined` data types should be pointers such as `Orderlist, Territory....` but `int` doesnt have to be a pointer. 

this `int Order::countOrderID = 0;        //start counter for orders at 0` was fine :)

Reply:
fuck